src/validation-errors-custom-attribute.ts

```diff
@@ -14,7 +14,7 @@ export interface RenderedError {
 @customAttribute('validation-errors')
 export class ValidationErrorsCustomAttribute implements ValidationRenderer {
 
-  public static inject() {
+  public static inject(): [typeof DOM.Element, Lazy<ValidationController>] {
```

Comment:
https://github.com/aurelia/dependency-injection/issues/183

Reply:
Closed the other concurrent PullRequest. This one's better.

Reply:
@EisenbergEffect